### ISSUE TITLE
Missing fontawesome icons in 5.0.0-rc.96

### ISSUE THREAD

**Issue by CaptainGlac1er** [formio/formio.js#5895]
**Describe the bug**
In the form builder a couple icons are missing. 
The remove button is currently trying to use fa-remove (which doesn't exist) maybe is suppose to use fa-trash?
The move button is currently trying to use fa-arrows (which only exists for PRO) maybe is suppose to use fa-arrows-alt for 5.x fontawesome or fa-arrows-up-down-left-right for fontawesome 6.x?

**Version/Branch**
5.0.0-rc.96

**To Reproduce**
Use: "@fortawesome/fontawesome-free": "^5.15.4"
1. Create form
2. Add component
3. Hover to see buttons in the top right

**Expected behavior**
See all buttons icons

**Screenshots**
![image](https://github.com/user-attachments/assets/bdc4e8c9-1f0c-41bb-a8b0-7236ef352bd4)

**Comment by lane-formio** [comment]
<div>Agent Viktoryia Alennikava linked Freshdesk ticket <a href="https://formhelp.freshdesk.com/a/tickets/53980">53980</a> for this issue.</div><br/>

**Comment by blakekrammes** [comment]
The form builder for `formio.js` 5.x [uses bootstrap 5 by default](https://github.com/formio/formio.js/blob/v5.0.0-rc.96/src/templates/index.js), which includes its own icon library (bootstrap icons) and does not use fontawesome. Although, it looks like we still have fontawesome 4 classes on our bootstrap 5 templates in addition to the bootstrap icon classes. You are welcome to take a look at our template repo and make a PR that supports fontawesome 5. https://github.com/formio/bootstrap

**Comment by CaptainGlac1er** [comment]
Ah so stuff like `fa-times` here is just legacy fontawesome 4? 
https://github.com/formio/bootstrap/blob/5a2451899db1ebf59fc09cd16fe60b5b566ffb11/src/templates/bootstrap5/address/form.ejs#L14 
What if you want to use bootstrap 5 classnames but use fontawesome for icons? If that were to be created would it be `bootstrap5-fontawesome5`?

**Comment by blakekrammes** [comment]
Yes. And while you could create a new global template, what is probably a better solution would be overriding the specific template(s) that are causing you grief: https://help.form.io/developers/form-development/form-templates#overriding-templates. 

And I believe this is the template you'd want to override https://github.com/formio/bootstrap/blob/5a2451899db1ebf59fc09cd16fe60b5b566ffb11/src/templates/bootstrap5/builderComponent/form.ejs
